notebooks/community/experiments/vertex_ai_model_experimentation.ipynb

```diff
@@ -0,0 +1,1750 @@
+{
```

Comment:
<p>Add a bit of text what each helper function does</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>done</p>